packages/components/src/tabs/tabs.tsx

```diff
@@ -163,15 +195,35 @@ const TriggerBase = styled(View, {
       },
     },
     active: {
-      true: {
-        cursor: 'default',
-        backgroundColor: '$neutral-50',
-      },
-      false: {
-        backgroundColor: '$neutral-10',
-        hoverStyle: {
-          backgroundColor: '$neutral-20',
-        },
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
I couldn't figure out how to fix this error in Tamagui, even though it's a valid syntax.

Reply:
Giving the function a type explicitly breaks type inference.